### ISSUE TITLE
Stackview has concrete height inside a modal

### ISSUE THREAD

**Issue by bbogdanov** [vmware-clarity/ng-clarity#265]
## Describe the bug

Hi guys,

https://github.com/vmware-clarity/ng-clarity/blob/5840b57ecd00bc16009bdbcae4cee418c8fb1865/projects/angular/src/data/stack-view/_stack-view.clarity.scss#L370

Don't know why that was set there but I believe it's a bug.

What I expected was to see the stack-view taking as much space it needs in its container no matter if it is in a modal or not.

What I experience is using a wizard > datagrid > detail pane > stack-view. Because of the `.modal & { height: 55vh;  margin-bottom: 0; }` the stack-view takes only `55vh` instead of as much size it needs in the detail pane. 

Maybe a more concrete selector or just removing the styles for the modal or letting the dev deciding what the height should be at certain point will be much better and by default to take the as much size it needs.

I'll try to provide a demo tomorrow.

## How to reproduce

1.  Add wizard instance
2.  Add a datagrid with detail pane inside a wizard step
3.  Add a stackview inside the detail pane of the datagrid
4.  Add several stackview blocks until you see the stackview scrollbar

## Expected behavior

Seeing the container that holds the stackview available for scroll instead of only able to scroll the stackview.
In other words - detail pane becoming scrollable instead of the stackview itself.

## Versions

**Clarity version:**

- [ ] v12.x
- [x] v13.x
- [ ] Other:

**Framework:**

- [x] Angular

**Framework version:**
Angular 14
